### PR TITLE
Shimmer themes

### LINKED ISSUE #1717: Package Request: Numix GTK Theme
https://github.com/shimmerproject/Numix

### PR DESCRIPTION
**Pull request by skrzyp**
Some of greatest GTK themes from [Shimmer Project](https://github.com/shimmerproject) - Numix and rest of family.

Fixes #1717

### CODE REVIEW

File: srcpkgs/albatross-themes/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+noarch=yes
+short_desc="An elegant black GTK2/3/Metacity/Xfwm theme"
+build_style=fetch
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+license="GPL-2"
+depends="gtk+"
+homepage="https://github.com/shimmerproject/Albatross"
+distfiles="https://github.com/shimmerproject/Albatross/archive/v${version}.tar.gz"
+checksum="e8900f1e603cae1f6c4f90146226e64e9700600878c515a6e39aae0a59c36cc7"
```
Comment (beefcurtains) [inline srcpkgs/albatross-themes/template]: I get ``ff40e28e164cb99f01b131ae3e79a07782e29a720535460e901ce305fa322ae1``

File: srcpkgs/blackbird-themes/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+noarch=yes
+short_desc="An elegant dark GTK2/3/Metacity/Xfwm theme"
+build_style=fetch
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+license="GPL-2"
+depends="gtk+"
+homepage="https://github.com/shimmerproject/Blackbird"
+distfiles="https://github.com/shimmerproject/Blackbird/archive/v${version}.tar.gz"
+checksum="1cabf8e621ee325a1ae32e370aa567158d0cec4c71298cc3b67b208e11d4df83"
```
Comment (beefcurtains) [inline srcpkgs/blackbird-themes/template]: I get ``ca31362254df2d336b2b090deb925f19a1dba72632ed9c7f82cf406be89ec1e6``

File: srcpkgs/bluebird-themes/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+noarch=yes
+short_desc="An elegant blue GTK2/3/Metacity/Xfwm theme"
+build_style=fetch
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+license="GPL-2"
+depends="gtk+"
+homepage="https://github.com/shimmerproject/Bluebird"
+distfiles="https://github.com/shimmerproject/Bluebird/archive/v${version}.tar.gz"
+checksum="e09e912834ff9ebe058ef0d8fb013c1af4e0adc8812e83c2b3d0022251d1075c "
```
Comment (beefcurtains) [inline srcpkgs/bluebird-themes/template]: I get ``f76520181d9cb74a0cf1c61728ab02a7fb5f0835fef036ebd68f1e89caa0c579``

File: srcpkgs/greybird-themes/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+noarch=yes
+short_desc="An elegant grey GTK2/3/Metacity/Xfwm theme"
+build_style=fetch
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+license="GPL-2"
+depends="gtk+"
+homepage="https://github.com/shimmerproject/Greybird"
+distfiles="https://github.com/shimmerproject/Greybird/archive/v${version}.tar.gz"
+checksum="cf134f5d3c3886161932ca7f7db30bad8a4aa9966421f0844017a669af1ed08c"
```
Comment (beefcurtains) [inline srcpkgs/greybird-themes/template]: I get ``983b6fe80ac8fb3598b01b53379c617b43a60762df6ea90bd4373fe2dec71fa5``

File: srcpkgs/numix-themes/template [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+noarch=yes
+short_desc="A modern flat GTK2/3/Metacity/Xfwm theme"
+build_style=fetch
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+license="GPL-3"
+depends="gtk+"
+homepage="https://github.com/shimmerproject/Numix"
+distfiles="https://github.com/shimmerproject/Numix/archive/v${version}.tar.gz"
+checksum="1b4d99014c838dd787afed5fb15e06948f43d7f917cfa6bf91a9ae446f2f4d10"
```
Comment (beefcurtains) [inline srcpkgs/numix-themes/template]: I get ``64b0c34c49633879c8b0b9b685da7b927501ab6db265db15fed04705c04f935b``

**Comment by beefcurtains** [comment]
Is it necessary to have the version number in the name under /usr/share/themes?

**Comment by skrzyp** [comment]
Maybe no, but I coudn't figure out to leave theme's names without version as root folder…

**Comment by skrzyp** [comment]
So… what's going on with this PR? Do you want to merge that?

**Comment by thypon** [comment]
There are "5 May" take it easy.

**Comment by xtraeme** [comment]
I'd like to see unversioned dirs in `/usr/share/themes` as @beefcurtains said above.

**Comment by skrzyp** [comment]
They've fixed right now, I'm sure that's a final iteration of rebasing, phew -.-

File: srcpkgs/blackbird-themes/template [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+license="GPL-2"
+depends="gtk+"
+homepage="https://github.com/shimmerproject/Blackbird"
+distfiles="https://github.com/shimmerproject/Blackbird/archive/v${version}.tar.gz"
+checksum="ca31362254df2d336b2b090deb925f19a1dba72632ed9c7f82cf406be89ec1e6"
+
+do_install() {
+	tar xzf v${version}.tar.gz
+	vmkdir usr/share/themes
+	vcopy Blackbird-${version} usr/share/themes/Blacbird
```
Comment (beefcurtains) [inline srcpkgs/blackbird-themes/template]: +k

**Pull request merged**